sphinx/api/command-reference.rst

```diff
@@ -16,21 +16,38 @@ Rose Commands
 rose config-edit
 ^^^^^^^^^^^^^^^^
 
-TODO: This is here to allow the documentation tests to pass
+.. warning::
+
+   The Rose Edit GUI has not yet been reimplemented in Rose 2.
+
+   The old Rose 2019 (Python 2) GUI remains compatible with Rose 2
+   configurations.
 
 .. _command-rose-suite-run:
 
 rose suite-run
 ^^^^^^^^^^^^^^
 
-TODO: This is here to allow the documentation tests to pass
+This command has been replaced by ``cylc install ; cylc play``.
```

Comment:
Refraining from making further comment on these sections given the `.. TODO: ` below. Happy to make suggestions now, or afterwards if you wish.

Reply:
I started making docs changes but backed out to do one thing at a time.

We need to remove `rose suite-run` references from the docs next :grimacing:.